generator/internal/genclient/translator/protobuf/testdata/scalar.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+package test;
+
+// A test message.
+message Fake {
+	// A singular field tag = 1
```

Comment:
nit: here and in the other files, the proto files need formatting now.

Reply:
ack